### DISCUSSION TITLE
Block diagram rendering broken in project's issue preview

### DISCUSSION THREAD

**Discussion by VinayBanakar** [community/community#38613] [Projects and Issues]
Hi, I noticed that block diagrams generated with [mermaid-js](https://github.com/mermaid-js/mermaid) do not render in the preview of issues in the project kanban. 

Example,
![image](https://user-images.githubusercontent.com/7550766/200698518-c3d2dab4-d5a2-413a-934b-2e8e7ba6bd75.png)

I can see that when the issue is opened in a new tab, the diagrams are rendered correctly.
![image](https://user-images.githubusercontent.com/7550766/200698775-1e9e3193-7200-4a70-af11-e61be7c3c726.png)

Can you please fix this?

**Comment by ossanna16** [comment]
Hi there @VinayBanakar and welcome to our community! Thank you for reporting this!

**Reply by wheres-baldo** [reply]
I just reported this issue to GitHub support - did y'all ever find a fix?
